### PR TITLE
s/Rem/Remove/g to improve code readability

### PR DESCRIPTION
**Pull request by pattyshack**
Rem doesn't save much typing but requires more brain power to read

### CODE REVIEW

File: engine/verification/fetcher/engine_test.go [hunk 1]
```diff
@@ -576,10 +576,10 @@ func mockPendingChunksAdd(t *testing.T, pendingChunks *mempool.ChunkStatuses, li
 // mockPendingChunksRem mocks the remove method of pending chunks for expecting only the specified list of chunk statuses.
```
Comment (SaveTheRbtz) [inline engine/verification/fetcher/engine_test.go:576]: comment

File: module/mempool/herocache/dns_cache_test.go [hunk 2]
```diff
@@ -225,7 +225,7 @@ func testAddToCache(t *testing.T,
 }
 
 // TestDNSCache_Rem checks the correctness of cache against removal.
```
Comment (SaveTheRbtz) [inline module/mempool/herocache/dns_cache_test.go:227]: comment also

File: module/mempool/stdmap/incorporated_result_seals_test.go [hunk 3]
```diff
@@ -130,7 +130,7 @@ func (m *icrSealsMachine) Rem(t *rapid.T) {
 }
 
 // RemUnknown is an action that removes an unknown element from the icrSeals
-// This mostly tests Rem has no insertion side-effects
+// This mostly tests Remove has no insertion side-effects
 func (m *icrSealsMachine) RemUnknown(t *rapid.T) {
```
Comment (SaveTheRbtz) [inline module/mempool/stdmap/incorporated_result_seals_test.go:134]: RemoveUnkown?

**Review by SaveTheRbtz — APPROVED** [review]
As someone who deeply hates the creat(2) I definitely approve this diff!

**Comment by pattyshack** [comment]
(I found and replace a bunch more.  In particular, RemIdFromKey)

**Comment by pattyshack** [comment]
bors merge

**Pull request merged**